index.html

```diff
@@ -66,7 +66,7 @@ <h2>Multi-Post Stories Gain+Glory</h2>
             <li>JavScript</li>
             <li>html</li>
```

Comment:
- [ ] OPTIONAL: I recommend making the popup smaller, because currently, the image looks too stretched: 
<img width="1367" alt="Screenshot 2022-03-03 at 12 02 00" src="https://user-images.githubusercontent.com/77038610/156522105-05f1e32f-bae6-45b7-93a0-5c3099073474.png">

Reply:
Yeah, sure